### ISSUE TITLE
ImportError: No module named cv

### ISSUE THREAD

**Issue by pittebidde** [nicknochnack/TFODCourse#143]
**TL:DR**:  If you get the Problem "ImportError: No module named cv2" or similar, it could be because your virtual enviroment does not yet have the library. you might need to install the needed libraries (opencv and numpy) locally in your virtual env. via command prompt.

I already solved this problem by myself, but since i haven't found any similar Issues in the "Issues Tab" yet, i might as well document my problem

When you create a virtual environment, your global libraries are not connected to your virtual environment setup.
So if you try to import them, they won't find the library.

My solution (i dont know if it is a particulary clean one), was to:
1. open the command promt
2. activate your virtual environment (in my case it was windows, so i typed in ".\tfod\Scripts\activate")
3. install the libraries locally (you need not only opencv, but also numpy because it is also used in opencv)
 meaning i typed into the virtual environment command line:
 pip install opencv-python
 and
 pip install numpy

**Comment by GunSY** [comment]
I am getting this error No module named 'numpy._utils'
I have uninstalled and then reinstalled numpy lib, but the errors are still there.
Pip list also showing I have the lib installed.
![error](https://github.com/nicknochnack/TFODCourse/assets/5249935/0e292228-9152-4a9c-892d-4b090a39517d)

![error-1](https://github.com/nicknochnack/TFODCourse/assets/5249935/4a6895da-06c2-4b3f-9d7e-1ac1462ce1d0)

**Comment by GunSY** [comment]
Can you let me know what has happened?
